src/FormBuilder/resources/js/storage.js

```diff
@@ -27,4 +27,19 @@ window.storage = {
             settings: JSON.parse( window.storageData.settings || "{}" ),
         };
     },
+    preview: ( blocks ) => {
```

Comment:
I would really love to see this file concerted to typescript as well so we can specify a type for the `blocks` param

Reply:
Let's address this in a new PR using a custom Type that can be shared throughout the application.

Reply:
Resolving in favor of https://github.com/impress-org/givewp-next-gen/issues/77.